src/plugins/factoids/storage.persistent.js

```diff
@@ -0,0 +1,34 @@
+let STORE;
+
+exports.getStore = () => {
+  // eslint-disable-next-line global-require
+  const { Store } = require('./storage.internal');
+  if (STORE) {
+    if (Object.getPrototypeOf(STORE) !== Store.prototype) {
+      Object.setPrototypeOf(STORE, Store.prototype);
```

Comment:
why is this needed?

Reply:
It's for HMR, especially in development. I know, it's weird, but I want to keep it.